pkg/logql/explain.go

```diff
@@ -0,0 +1,59 @@
+package logql
+
+import (
+	"github.com/xlab/treeprint"
```

Comment:
So I'd prefer [treeprinter](https://github.com/cockroachdb/cockroach/blob/master/pkg/util/treeprinter/tree_printer.go.) from CockroachDB but it's under a business license. It was under Apache before. I need to check with legal if we could copy it before it changed.